scripts/docs_version_pragma_check.sh

```diff
@@ -65,7 +65,7 @@ function getAllAvailableVersions()
 {
     allVersions=()
     local allListedVersions=( $(
-        wget -q -O- https://ethereum.github.io/solc-bin/bin/list.txt |
+        wget -q -O- https://binaries.soliditylang.org/bin/list.txt |
```

Comment:
This was still using the deprecated solc-bin URLs so snippets requiring versions >= 0.7.2 were triggering `No release ... was listed in available releases!`...